<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <widget class="QTabWidget" name="tabWidget_menu">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_controller_2">
       <attribute name="title">
        <string>Controller</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget_5">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>692</width>
          <height>504</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <widget class="QLabel" name="AIR_2">
           <property name="font">
            <font>
             <pointsize>18</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="pixmap">
            <pixmap resource="recource.qrc">:/icon/主机.jpg</pixmap>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <pointsize>22</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>central air-condition</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_setmodel">
           <property name="title">
            <string>Model</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_11">
                  <item>
                   <widget class="QLabel" name="label_model_2">
                    <property name="font">
                     <font>
                      <pointsize>11</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>模式</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="comboBox_model">
                    <item>
                     <property name="text">
                      <string>制冷</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>制热</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout_13">
                  <item>
                   <widget class="QLabel" name="label_4">
                    <property name="font">
                     <font>
                      <pointsize>11</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>温度范围</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="comboBox_mintem">
                    <item>
                     <property name="text">
                      <string>16℃</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>17℃</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>18℃</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>19℃</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>20℃</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLabel" name="label_between_2">
                    <property name="text">
                     <string>～</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QComboBox" name="comboBox_maxtem">
                    <item>
                     <property name="text">
                      <string>21℃</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>22℃</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>23℃</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>24℃</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>25℃</string>
                     </property>
                    </item>
                    <item>
                     <property name="text">
                      <string>26℃</string>
                     </property>
                    </item>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_12">
                <item>
                 <widget class="QLabel" name="label_wind_2">
                  <property name="font">
                   <font>
                    <pointsize>11</pointsize>
                   </font>
                  </property>
                  <property name="text">
                   <string>风速</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_wind">
                  <item>
                   <property name="text">
                    <string>低速</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>中速</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>高速</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_dispatch">
                  <property name="text">
                   <string>调度策略</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_dispatch">
                  <item>
                   <property name="text">
                    <string>风速、服务时间排序</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>时间片轮转</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_14">
                <item>
                 <spacer name="horizontalSpacer_7">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
                <item>
                 <widget class="QPushButton" name="putton_setmodel">
                  <property name="text">
                   <string>确认</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_8">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_setprice">
           <property name="title">
            <string>Price</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QLabel" name="label_price_2">
              <property name="text">
               <string>单位电价</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <spacer name="horizontalSpacer_9">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>176</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="2">
             <spacer name="horizontalSpacer_10">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>176</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="0" column="3">
             <widget class="QLineEdit" name="lineEdit_price_2"/>
            </item>
            <item row="0" column="4">
             <widget class="QLabel" name="label_price_1">
              <property name="text">
               <string>元/kW*h</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="5">
             <layout class="QHBoxLayout" name="horizontalLayout_16">
              <item>
               <spacer name="horizontalSpacer_11">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QPushButton" name="putton_setprice">
                <property name="text">
                 <string>确认</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_12">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_monitor_2">
       <attribute name="title">
        <string>Monitor</string>
       </attribute>
       <widget class="QWidget" name="layoutWidget_8">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>50</y>
          <width>741</width>
          <height>431</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_17">
           <item>
            <widget class="QTextBrowser" name="textBrowser_room1"/>
           </item>
           <item>
            <widget class="QTextBrowser" name="textBrowser_room2"/>
           </item>
           <item>
            <widget class="QTextBrowser" name="textBrowser_room3"/>
           </item>
           <item>
            <widget class="QTextBrowser" name="textBrowser_room4"/>
           </item>
           <item>
            <widget class="QTextBrowser" name="textBrowser_room5"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_18">
           <item>
            <widget class="QLabel" name="label_roomnum">
             <property name="text">
              <string>房间号</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lineEdit_roomnum"/>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_print">
             <property name="text">
              <string>结账</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_showlog">
             <property name="text">
              <string>查看报表</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_exit">
      <property name="text">
       <string>退出系统</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="recource.qrc"/>
 </resources>
 <connections/>
</ui>
